packages/manager/apps/pci-private-network/src/api/utils/utils.spec.ts

```diff
@@ -85,3 +87,33 @@ describe('getAutoGeneratedName', () => {
     expect(result).toMatch(/^\d+-\d+$/);
   });
 });
+
+describe('isValidCidrMask', () => {
+  it.each([0, 33])(
+    'should return false when number is not between 1 and 32',
+    (cidr) => {
+      const result = isValidCidrMask(cidr);
+      expect(result).toBe(false);
+    },
+  );
+
+  it('should return true when number is between 1 and 32', () => {
+    const result = isValidCidrMask(16);
+    expect(result).toBe(true);
+  });
```

Comment:
Just for information, you can even have a dynamic test to keep your level of detail 
```suggestion
  it.each([
  [false, 0],
  [false, 33],
  [true, 16]
  ])(
    'should return %s when number is %s',
    (isValidCidr, cidr) => {
      const result = isValidCidrMask(cidr);
      expect(result).toBe(isValidCidr);
    },
  );

```